templates/lightning/layout.html

```diff
@@ -3,4 +3,10 @@
 {% block header %}
 <!-- Socket.IO JS -->
 <script src="{{ url_for('static',filename='assets/lightning/socket.io.js') }}"></script>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
+
+{% block content %}
```

Comment:
this layout template should not have a content block

Reply:
you should put a new block in the main layout template called ```navigation-bar``` or something

Reply:
this is the result when i add a navbar:
![image](https://user-images.githubusercontent.com/47799652/165648664-b8f75fa9-70b5-4c5e-a426-5690d7d06853.png)

its basically creating one new navbar on top of the main navbar. not good.

Reply:
ill delete the button from the lightning/layout and put it on individual pages instead